code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -214,6 +214,7 @@
 	else
 		icon_state = "[initial(icon_state)]0"
 
+/* Replaced with variant in whitesands/code/modules/reagents/reagent_containers/hypospray.dm
```

Comment:
You can completely remove this if you want. 
We're not keeping modularity as a design core as it has major problems in the future.

Reply:
ok, by reviewing the rest of this you can probably tell I gave up on modularity too